compressor/base.py

```diff
@@ -260,7 +274,7 @@ def precompile(self, content, kind=None, elem=None, filename=None,
 
         filter_or_command = self.precompiler_mimetypes.get(mimetype)
         if filter_or_command is None:
-            if mimetype in ("text/css", "text/javascript"):
+            if mimetype in self.output_mimetypes:
```

Comment:
...funny, almost looks like it did allow css files in an `compress js` block (and vice versa)...

Reply:
Yeah, I assumed that was an oversight and changed it to something simpler.